pages/download.html

```diff
@@ -16,20 +16,20 @@
 unstable_ubuntu64_analytics_conversion: /downloads/2.1.0-rc1-ubuntu64
 unstable_ubuntu_ppa_analytics_conversion: /downloads/2.1.0-rc1-ubuntu-ppa
 stable_short_version: Mixxx 2.1
```

Comment:
This needs to be updated to 2.1.1 too.

Reply:
Why? It is still a 2.1 Version. We have also not had the minor version before.

Reply:
We should have had the full version number before!

Reply:
It is inconsistent to have 2.1 in some places but 2.1.1 in others.